### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by slackmcgrath**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,4 @@ Testing as assignee
 
 adding as reviewer
 
+1.
```
Comment (MikaelaMcGrath) [inline README.md:14]: should be
```suggestion
13
```

**Comment by MikaelaMcGrath** [comment]
Looks good

File: README.md (hunk 1 above)
Comment (slackmcgrath) [reply README.md:14]: creating a thread

Comment (MikaelaMcGrath) [reply README.md:14]: yeah I like this

**Comment by MikaelaMcGrath** [comment]
@slackmcgrath work harder

**Pull request merged**